modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -1285,3 +1285,103 @@ def finalize(cls, partitions):
             Partitions of Modin Dataframe on which all deferred calls should be performed.
         """
         [part.drain_call_queue() for row in partitions for part in row]
+
+    @classmethod
+    def repartition(cls, partitions, axis, lengths, widths):
```

Comment:
shouldn't you implement `.repartition()` in other partition managers (omnisci maybe?)

Reply:
The implementation is fairly generic and works for both Ray and Dask. As for OmniSci, `concat` method is overridden for it, from which `repartition` is called, so no need to implement it here.

This is also not relevant for OmniSci because the dataframe consists of only one partition.